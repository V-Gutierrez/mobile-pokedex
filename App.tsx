import React from 'react'
import {
  SafeAreaView,
  StatusBar,
  StyleSheet,
  Text,
} from 'react-native'



const styles = StyleSheet.create({})

const App = () => {
  return (
    <SafeAreaView >
      <StatusBar />
      <Text>Template</Text>
    </SafeAreaView>
  )
}


export default App
