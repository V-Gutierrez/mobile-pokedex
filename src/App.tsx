import { Header } from './components'
import { PokemonList } from './views'
import React from 'react'
import {
  SafeAreaView,
  StyleSheet,
} from 'react-native'


const styles = StyleSheet.create({
  container: {
    flex: 1,
  }
})

const App = () => {
  return (
    <SafeAreaView style={styles.container}>
      <Header />
      <PokemonList />
    </SafeAreaView>
  )
}


export default App
