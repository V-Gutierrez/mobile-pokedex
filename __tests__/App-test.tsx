import 'react-native'
import React from 'react'
import App from '../src/App'

import renderer from 'react-test-renderer'

describe('App component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<App />)
    expect(tree).toMatchSnapshot()
  })
})
