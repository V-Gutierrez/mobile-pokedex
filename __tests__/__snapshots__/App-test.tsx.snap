// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "height": "25%",
        "justifyContent": "space-between",
        "paddingHorizontal": 10,
        "paddingVertical": 20,
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 24,
            "justifyContent": "center",
            "marginLeft": 10,
          }
        }
      >
        Mobile Pokedex
      </Text>
    </View>
    <View>
      <Image
        source={
          Object {
            "testUri": "../../../src/assets/images/pokeball_background.png",
          }
        }
        style={
          Object {
            "marginLeft": 75,
            "opacity": 0.2,
            "zIndex": -1,
          }
        }
      />
    </View>
  </View>
</RCTSafeAreaView>
`;
